### PR TITLE
Gccurtis: Data page displays name

### PR DESCRIPTION
**Pull request by gccurtis**
Implemented the changes in Week 3 step 1, Data page now shows name.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/Data2Servlet.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@WebServlet("/data2")
+public class Data2Servlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String input = request.getParameter("name");
+    response.setContentType("text;");
```
Comment (ohfunc) [inline portfolio/src/main/java/com/google/sps/servlets/Data2Servlet.java]: Should this be text/plain? I'm not sure if content types can include the type without the subtype.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getData(){
+	const name = document.getElementById('text-input').value; 
+	fetch('/data2?name='+name).then(response => response.text()).then((data) => {
+	document.getElementById('data-display').innerText = data;
```
Comment (ohfunc) [inline portfolio/src/main/webapp/script.js]: Code within anonymous functions should be indented.

File: portfolio/src/main/java/com/google/sps/servlets/Data2Servlet.java (hunk 1 above)
Comment (gccurtis) [reply portfolio/src/main/java/com/google/sps/servlets/Data2Servlet.java]: I believe you are correct, so I pushed with it fixed.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (gccurtis) [reply portfolio/src/main/webapp/script.js]: Alrighty, so I pushed with it fixed.

**Review by ohfunc — COMMENTED** [review]
Hi Gabriel! I don't see the fixes addressed from the last round of comments. Maybe we can discuss offline?

**Comment by gccurtis** [comment]
You were right I pushed on the wrong branch, all should be good now.

**Review by Howon — APPROVED** [review]

**Pull request merged**